sensors/aidl/Sensor.cpp

```diff
@@ -122,7 +125,7 @@ void Sensor::run() {
             });
         } else {
             timespec curTime;
-            clock_gettime(CLOCK_REALTIME, &curTime);
+            clock_gettime(CLOCK_BOOTTIME, &curTime);
```

Comment:
Add reason for changing it to Boot time in commit message

Reply:
google is recommending CLOCK_BOOTTIME to calculate current time as it keeps running during suspend.